### PR TITLE
Removes unused imports and fixes bad indentations 

### PR DESCRIPTION
**Pull request by AHAAAAAAA**
Removed unused imports and did some indentation cleanups. Trying to make landscape less unhappy with us.

## Description

A lot of unused imports in a lot of files removed. Fixed indentation in 2-3 files.

## Motivation and Context

Code style consistency, better practice.

## How Has This Been Tested?

Runs, haven't seen any bugs thus far.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/models.py [hunk 1]
```diff
@@ -252,23 +251,23 @@ def parse_map(map_dict, iteration_num, step, step_location):
...
+                    pokestops[f['id']] = {
+                        'pokestop_id': f['id'],
+                        'enabled': f['enabled'],
+                        'latitude': f['latitude'],
+                        'longitude': f['longitude'],
+                        'last_modified': datetime.utcfromtimestamp(
+                            f['last_modified_timestamp_ms'] / 1000.0),
+                        'lure_expiration': lure_expiration,
+                        'active_pokemon_id': active_pokemon_id
+                }
```
Comment (relrod) [inline pogom/models.py]: This `}` needs one more indent for consistency.

**Comment by medghaim** [comment]
Testing

**Comment by medghaim** [comment]
+1

**Comment by MonkeyLeeT** [comment]
Tested, no crashing or anything. Frontend working. +1

**Pull request merged**